addons/hr_calendar/static/src/calendar/calendar_model.js

```diff
@@ -0,0 +1,42 @@
+/** @odoo-module **/
+
+import { AttendeeCalendarModel } from "@calendar/views/attendee_calendar/attendee_calendar_model";
+import { serializeDate } from "@web/core/l10n/dates";
+import { patch } from "@web/core/utils/patch";
+patch(AttendeeCalendarModel.prototype, {
+    setup () {
+        super.setup(...arguments)
+        this.data.workingHours = {};
+    },
+
+    get workingHours() {
+        return this.data.workingHours;
+    },
+
+    get employeeId() {
+        return this.meta.context.employee_id && this.meta.context.employee_id[0] || null;
+    },
+
+    async updateData(data) {
+        await super.updateData(...arguments)
+        data.workingHours = await this.fetchWorkingHours(data);
+    },
+
+    async fetchWorkingHours(data){
+        if ((data.range.end - data.range.start)/(1000*3600*24) >= 28) {
```

Comment:
what are we checking here? :eyes:

Reply:
It was a bad way to check the calendar's view. :sweat: